sycl/test/lit.cfg.py

```diff
@@ -112,15 +112,15 @@
 if triple == 'nvptx64-nvidia-cuda':
     config.available_features.add('cuda')
 
+
 if triple == 'amdgcn-amd-amdhsa':
     config.available_features.add('hip_amd')
-    # For AMD the specific GPU has to be specified with --offload-arch
-    if not re.match('.*--offload-arch.*', config.sycl_clang_extra_flags):
-        raise Exception("Error: missing --offload-arch flag when trying to "  \
-                        "run lit tests for AMD GPU, please add "              \
-                        "--hip-amd-arch=<target> to buildbot/configure.py or add"  \
-                        "`-Xsycl-target-backend=amdgcn-amd-amdhsa --offload-arch=<target>` to " \
-                        "the CMake variable SYCL_CLANG_EXTRA_FLAGS")
+    # For AMD the specific GPU has to be specified with --offload-arch, only
+    # compiler tests are run so hardcode the offload arch to gfx908
```

Comment:
Don't developers want/need to test compilation for other AMD GPU architectures?
I thought that patch removes the requirement by setting the default rather than removing the customization option completely.

Reply:
Oh that's a good point, it would make the patch simpler.

Although given the nature of the lit tests I think it's unlikely for the AMD GPU architecture to make much of a difference. For example we also don't really have a similar flag to test multiple SM version with the CUDA backend, although it can still be done using the `SYCL_CLANG_EXTRA_FLAGS`.

What would you prefer:
* Remove the option completely (current patch)
* Remove the buildbot option but make sure it can still be configured manually through `SYCL_CLANG_EXTRA_FLAGS`
* Keep the buildbot option and just add the default

Personally I would lean towards the second point as I think it's going to be a fairly uncommon use case and that the buildbot option is kind of confusing but I don't really mind either way.

Reply:
> Personally I would lean towards the second point as I think it's going to be a fairly uncommon use case and that the buildbot option is kind of confusing but I don't really mind either way.

Sounds good to me.
On the other hand, I do not develop for AMD GPU, so my preference/opinion shouldn't be considered for this question. I might be wrong here and "removing the option completely" (current patch) is good enough for developers working on enabling AMD GPU support. :)
Just let me know, which option you are going with.

Reply:
Updated with the second option, so it can still be tweaked from the CMake.

Reply:
Great. Thanks.